src/Processor/ProductProcessor.php

```diff
@@ -218,6 +224,16 @@ public function process(array $data): void
                 $productVariant->setName($translation['Name']);
             }
 
+            foreach ($variantData['ChannelPricings'] as $channelCode => $channelPricingData) {
+                /** @var ChannelPricingInterface $channelPricing */
+                $channelPricing = $this->channelPricingFactory->createNew();
+                $channelPricing->setChannelCode($channelCode);
+                $channelPricing->setPrice($channelPricingData['Price']);
+                $channelPricing->setOriginalPrice($channelPricingData['OriginalPrice']);
+
+                $productVariant->addChannelPricing($channelPricing);
```

Comment:
It might fail on subsequent imports because product variant is reused (and when a channel pricing exists for a given channel, addChannelPricing does nothing and fails to add it silently). We should reuse the existing channel pricing if possible.

Reply:
Doesn't matter, we clear the database before importing it anyway.